safeeyes/plugins/trayicon/plugin.py

```diff
@@ -76,10 +76,7 @@ def __init__(self, context, plugin_config):
         self.menu = Gtk.Menu()
 
         # Next break info menu item
-        self.item_info = Gtk.ImageMenuItem()
-        img_timer = Gtk.Image()
-        img_timer.set_from_icon_name("io.github.slgobinath.SafeEyes-timer", 16)
-        self.item_info.set_image(img_timer)
+        self.item_info = Gtk.MenuItem()
```

Comment:
Unfortunately, gtk4 completely removes the option of having a MenuItem with an image.
However, in https://github.com/slgobinath/SafeEyes/pull/558 I propose to drop this code entirely.